Makefile

```diff
@@ -4,12 +4,16 @@ dynamic: lib
 	$(AR) -x libmpfun90.a
 	$(CXX) -shared *.o -o libmpfun90.so
 
-	
+
 lib: libmpfun90.a
 
 
-FC = gfortran
-FFLAGS = -fno-automatic -O2 -funroll-all-loops -std=legacy -fPIC
+FC := $(shell echo $$FC)
+ifndef FC
+	FC := gfortran
+endif
```

Comment:
What is the benefit of this over `FC ?= gfortran`? I have some alternative modifications in mind to shrink the makefile a bit more.

Reply:
ah, it defaults to `f77` then...

Reply:
Yeah, `FC ?= gfortran` should work, so I can just do that.